react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -1010,6 +1010,8 @@ exports[`MuiCozyTheme/ExpansionPanel should render examples: MuiCozyTheme/Expans
 
 exports[`MuiCozyTheme/Menus should render examples: MuiCozyTheme/Menus 1`] = `"<div></div>"`;
 
+exports[`MuiCozyTheme/Switch should render examples: MuiCozyTheme/Switch 1`] = `"<div></div>"`;
```

Comment:
Should we consider this as a valid snapshot? I would prefer no tests than an empty snapshot like this which is a false positive (same remark for all MUI snapshots in fact)